src/engraving/io/mscreader.cpp

```diff
@@ -119,13 +119,16 @@ QByteArray MscReader::readStyleFile() const
 
 QByteArray MscReader::readScoreFile() const

```

Comment:
what about another methods?
excerptNames
imageFileNames

Reply:
Good catch, I improved the solution